lib/formatter.js

```diff
@@ -23,18 +23,15 @@ export default {
   // formats the given text editor
   formatTextEditor(editor, range = null) {
     try {
-      const { status, stdout, stderr } = this.runFormat(
+      const { status, stdout, stderr, error } = this.runFormat(
         this.getTextInRange(editor, range)
       );
 
-      switch (status) {
-      case 0: {
+      if (status == 0 && !stderr.length && !error) {
```

Comment:
`stderr` can null, so we have to check it first before calling `length`. Maybe this should be:

`if (status == 0 && !error) {`

Reply:
@rgreenjr i have updated the code so that it will handle properly the case when `stderr` is null, and also replaced `error.message` with just `error`.